x-pack/plugins/secops/public/components/timeline/index.tsx

```diff
@@ -96,6 +101,17 @@ class StatefulTimelineComponent extends React.PureComponent<Props> {
       updateRange!({ id, range: selectedRange });
     };
 
+    const onToggleDataProviderEnabled: OnToggleDataProviderEnabled = ({
+      dataProvider,
+      enabled,
+    }) => {
+      window.alert(
```

Comment:
NOTE: I did not plumb through redux and from redux back into the data providers with this PR since we have potential upcoming redux re-plumbing. Instead I added this `window.alert` for the click back handler as a reminder.